nil pointer dereference when using shoutrrr with pushover

**Describe the bug**
Getting an error trying to send notifications with shoutrrr on startup. See logs and container configuration below.

**To Reproduce**
Steps to reproduce the behavior:
1. Run docker container.
2. Get error.

Container:
```
docker run \
        --name=watchtower \
        --env=WATCHTOWER_NOTIFICATIONS=shoutrrr \
        --env=WATCHTOWER_NOTIFICATION_URL=pushover://REDACTED/REDACTED/iPhone \
        --volume=/var/run/docker.sock:/var/run/docker.sock \
        --restart=unless-stopped \
        --detach=true \
        containrrr/watchtower:latest
```

**Expected behavior**
Should not error.

**Environment**
 - Platform: ubuntu server 18.04
 - Architecture: x86-64
 - Docker version: 19.03.8, build afacb8b7f0

**Logs from running watchtower with the `--debug` option**

```

goroutine 25 [running]:

github.com/containrrr/shoutrrr/pkg/router.(*ServiceRouter).Send(0x0, 0xc0001a82a0, 0x5d, 0x0, 0x7, 0x0, 0xc000118980)

	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200404203330-157bd996ea13/pkg/router/router.go:52 +0x3a

github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries.func1(0xc0002677a0, 0xc0001a82a0, 0x5d)

	/home/circleci/repo/pkg/notifications/shoutrrr.go:58 +0x67

created by github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries

	/home/circleci/repo/pkg/notifications/shoutrrr.go:57 +0x7f

time="2020-05-26T15:05:25Z" level=info msg="Starting Watchtower and scheduling first run: 2020-05-26 15:10:25 +0000 UTC m=+301.036760807"

panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xa16e2a]


goroutine 18 [running]:

github.com/containrrr/shoutrrr/pkg/router.(*ServiceRouter).Send(0x0, 0xc000190cc0, 0x5d, 0x0, 0x0, 0x0, 0x0)

	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200404203330-157bd996ea13/pkg/router/router.go:52 +0x3a

github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries.func1(0xc00038c960, 0xc000190cc0, 0x5d)

	/home/circleci/repo/pkg/notifications/shoutrrr.go:58 +0x67

created by github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries

	/home/circleci/repo/pkg/notifications/shoutrrr.go:57 +0x7f
```

Same problem here:
```
goroutine 12 [running]:
github.com/containrrr/shoutrrr/pkg/router.(*ServiceRouter).Send(0x0, 0xc000152300, 0x7a, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200404203330-157bd996ea13/pkg/router/router.go:52 +0x3a
github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries.func1(0xc0000aea80, 0xc000152300, 0x7a)
	/home/circleci/repo/pkg/notifications/shoutrrr.go:58 +0x67
created by github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries
	/home/circleci/repo/pkg/notifications/shoutrrr.go:57 +0x7f
time="2020-05-28T10:22:18+02:00" level=info msg="Starting Watchtower and scheduling first run: 2020-05-28 10:27:18 +0200 CEST m=+300.720956816"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xa16e2a]

goroutine 14 [running]:
github.com/containrrr/shoutrrr/pkg/router.(*ServiceRouter).Send(0x0, 0xc000414180, 0x7a, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200404203330-157bd996ea13/pkg/router/router.go:52 +0x3a
github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries.func1(0xc00026d980, 0xc000414180, 0x7a)
	/home/circleci/repo/pkg/notifications/shoutrrr.go:58 +0x67
created by github.com/containrrr/watchtower/pkg/notifications.(*shoutrrrTypeNotifier).sendEntries
	/home/circleci/repo/pkg/notifications/shoutrrr.go:57 +0x7f
```

docker-compose.yml:
```
---
version: "2"

services:
  watchtower:
    image: containrrr/watchtower
    container_name: watchtower
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    environment:
      - "TZ=Europe/Zurich"
      - "WATCHTOWER_CLEANUP=true"
      - "WATCHTOWER_NOTIFICATIONS=shoutrrr"
      - "WATCHTOWER_NOTIFICATION_URL=pushover://REDACTED/REDACTED/iPhone"
      - "WATCHTOWER_NOTIFICATION_TEMPLATE={{range .}}{{.Time.Format \"2006-01-02 15:04:05\"}} ({{.Level}}): {{.Message}}{{println}}{{end}}"
    restart: unless-stopped
```

--- a/pkg/notifications/notifier.go
+++ b/pkg/notifications/notifier.go
@@ -47,7 +47,9 @@ func NewNotifier(c *cobra.Command) *Notifier {
 		default:
 			log.Fatalf("Unknown notification type %q", t)
 		}
-		n.types = append(n.types, tn)
+		if tn != nil {
+			n.types = append(n.types, tn)
+		}
 	}
 
 	return n

--- a/pkg/notifications/shoutrrr.go
+++ b/pkg/notifications/shoutrrr.go
@@ -30,7 +30,11 @@ func newShoutrrrNotifier(c *cobra.Command, acceptedLogLevels []log.Level) t.Noti
 	flags := c.PersistentFlags()
 
 	urls, _ := flags.GetStringArray("notification-url")
-	r, _ := shoutrrr.CreateSender(urls...)
+	r, err := shoutrrr.CreateSender(urls...)
+	if err != nil {
+		fmt.Printf("Failed to initialize Shoutrrr notifications: %s\n", err.Error())
+		return nil
+	}
 
 	n := &shoutrrrTypeNotifier{
 		Urls:      urls,
@@ -52,8 +56,10 @@ func (e *shoutrrrTypeNotifier) buildMessage(entries []*log.Entry) string {
 }
 
 func (e *shoutrrrTypeNotifier) sendEntries(entries []*log.Entry) {
-	// Do the sending in a separate goroutine so we don't block the main process.
+
 	msg := e.buildMessage(entries)
+
+	// Do the sending in a separate goroutine so we don't block the main process.
 	go func() {
 		errs := e.Router.Send(msg, nil)
 

